htdocs/core/lib/signature.lib.php

```diff
@@ -146,6 +146,21 @@ function getOnlineSignatureUrl($mode, $type, $ref = '', $localorexternal = 1)
 		} else {
 			$out .= '&securekey='.dol_hash($securekeyseed.$type.$ref.(isModEnabled('multicompany') ? (is_null($object) ? '' : $object->entity) : ''), '0');
 		}
+	} else {
+		$securekeyseed = getDolUserString(dol_strtoupper($type).'ONLINE_SIGNATURE_SECURITY_TOKEN');
```

Comment:
Are you sure about getDolUserString ?
Not getDolGlobalString ?

Reply:
@eldy, yes don't know why, autocompletion of IDE put this first on this file, has it work I didn't see it. I've also change all other $conf->global in this page